service/application/src/test/java/com/solace/maas/ep/event/management/agent/plugin/route/handler/base/AsyncRoutePublisherImplTests.java

```diff
@@ -44,7 +45,10 @@ public class AsyncRoutePublisherImplTests {
     public void testStart() {
         AsyncWrapper asyncWrapper = mock(AsyncWrapper.class);
 
-        Exchange exchange = ExchangeBuilder.anExchange(mock(ExtendedCamelContext.class))
+        CamelContext cc = mock(CamelContext.class);
```

Comment:
ExtendedCamelContext previously extend CamelContext. In the 3 -> 4 camel migration they broker this, so now the ExtendedCamelContext is a member of the CamelContext. Needed to update the tests to reflect this.